lib/que/job.rb

```diff
@@ -77,7 +77,7 @@ def enqueue(*args)
         end
 
         if Que.mode == :sync && !t
-          run(*attrs[:args])
+          run(*JSON.parse(attrs[:args].to_json))
```

Comment:
@lawrencejones This is still not quite consistent. `JSON.generate` is the opposite of `JSON.parse`. Those two ensure valid JSON documents. `to_json` will happily produce JSON fragments.